src/services/findAllReferences.ts

```diff
@@ -760,6 +761,14 @@ function getPrefixAndSuffixText(entry: Entry, originalNode: Node, checker: TypeC
         }
     }
 
+    // If the node is a numerical indexing literal, then add quotes around the property access.
+    if (entry.kind !== EntryKind.Span && isNumericLiteral(entry.node) && isAccessExpression(entry.node.parent)) {
+        return {
+            prefixText: "'",
+            suffixText: "'"

```

Comment:
Use `getQuoteFromPreference(getQuotePreference(...))`

Reply:
Hmm that requires changing the signature of `findRenameLocations` in the `LanguageService` interface. Is that okay?

Reply:
Yes, you can add an extra optional parameter.